.github/workflows/detect-breaking-changes-report.yml

```diff
@@ -116,7 +121,9 @@ jobs:
     # Posts a notification to Slack if a PR has a breaking change and it did not have a breaking change before
     - name: Post to Slack
       id: slack
-      if: steps.levitate-run.outputs.exit_code == 1 && steps.does-label-exist.outputs.result == 0 && steps.levitate-run.outputs.shouldSkip != 'true'
+      env:
```

Comment:
This is going to break the workflow (as env is now defined twice, check the error message [here](https://github.com/check-spelling/grafana/actions/runs/4130891051)). 

Just merge the two `env` declarations, (this and the one at line `137`).